framework/test_app/runners/k8s/k8s_xds_client_runner.py

```diff
@@ -158,8 +159,19 @@ def run(  # pylint: disable=arguments-differ
             config_mesh=config_mesh,
             generate_mesh_id=generate_mesh_id,
             print_response=print_response,
+            enable_csm_observability=enable_csm_observability,
         )
 
+        # Create a PodMonitoring resource if CSM Observability is enabled
+        # This is GMP (Google Managed Prometheus)
+        if enable_csm_observability:
+            self._create_pod_monitoring(
```

Comment:
assign this to the class var, then follow this example:

https://github.com/grpc/psm-interop/blob/ea4f7eb127bdd900042de921cdf2a54321ce63e0/framework/test_app/runners/k8s/gamma_server_runner.py#L256-L260

→

https://github.com/grpc/psm-interop/blob/ea4f7eb127bdd900042de921cdf2a54321ce63e0/framework/test_app/runners/k8s/k8s_base_runner.py#L581-L586

→

https://github.com/grpc/psm-interop/blob/ea4f7eb127bdd900042de921cdf2a54321ce63e0/framework/infrastructure/k8s.py#L610-L620

Reply:
Now the cleanup is added. There are a couple more places I need to add but was apparent enough.